metron-analytics/metron-statistics/src/test/java/org/apache/metron/statistics/OnlineStatisticsProviderTest.java

```diff
@@ -96,19 +98,22 @@ private void validateEquality(Iterable<Double> values) {
     validateStatisticsProvider(aggregatedProvider, summaryStats, stats);
   }
 
-  @Test(expected = IllegalStateException.class)
+  @Test
   public void testOverflow() {
     OnlineStatisticsProvider statsProvider = new OnlineStatisticsProvider();
-    statsProvider.addValue(Double.MAX_VALUE + 1);
+    assertThrows(IllegalStateException.class, () -> statsProvider.addValue(Double.MAX_VALUE + 1));
   }
 
-  @Test(expected = IllegalStateException.class)
+  @Test
   public void testUnderflow() {
     OnlineStatisticsProvider statsProvider = new OnlineStatisticsProvider();
     double d = 3e-305;
-    for(int i = 0;i < 5;++i,d/=100000) {
-      statsProvider.addValue(d);
-    }
+//    for(int i = 0;i < 5;++i,d/=100000) {
+    assertThrows(IllegalStateException.class, () -> statsProvider.addValue(d));
+//    }
+//    for(int i = 0;i < 5;++i,d/=100000) {
```

Comment:
Looks like we changed the test semantics a bit here. Are we sure those loops should be removed? If so, can we remove the commented out code?

Reply:
ahh, I left that in, and forgot about it. That test actually throws on the first iteration of the loop anyway. I remember being surprised about it, because I wasn't really looking forward to figuring out how to fix it properly.

Dropped it, and fixed the error message in the original class (it actually says it's double overflow instead of underflow)